ironfish/src/primitives/blockheader.ts

```diff
@@ -180,27 +178,27 @@ export class BlockHeader {
     sequence: number,
     previousBlockHash: BlockHash,
     noteCommitment: NoteEncryptedHash,
-    nullifierCommitment: { commitment: NullifierHash; size: number },
     transactionCommitment: Buffer,
     target: Target,
     randomness = BigInt(0),
     timestamp: Date | undefined = undefined,
     graffiti: Buffer,
     noteSize?: number | null,
     work = BigInt(0),
+    nullifierSize?: number | null,
     hash?: Buffer,
   ) {
     this.sequence = sequence
     this.previousBlockHash = previousBlockHash
     this.noteCommitment = noteCommitment
-    this.nullifierCommitment = nullifierCommitment
     this.transactionCommitment = transactionCommitment
     this.target = target
     this.randomness = randomness
     this.timestamp = timestamp || new Date()
     this.graffiti = graffiti
     this.noteSize = noteSize ?? null
     this.work = work
+    this.nullifierSize = nullifierSize ?? null
```

Comment:
This is a trivial change, but I think `nullifierSize` should be next to `noteSize` here and in the parameters as well, since they're next to each other as properties above the constructor.

Reply:
Yea I thought about this. I put them away from each other so someone doesn't accidentally mix them up when using the constructor. I think the real fix for this like we discussed is to have named params for the constructor